README.md

```diff
@@ -209,3 +209,25 @@ Go to the "Racks" section, click the blue cog icon near your moved rack and in t
 This step is only necessary for racks you move into the console.
 
 Follow the steps in our [docs](https://docs.convox.com/management/console-rack-management/) - Moving an AWS Rack section to give the console role permission to access the EKS rack cluster.
+
+## Updating
```

Comment:
```suggestion
## Updating (V2 racks only)
```

Reply:
These are instructions regarding the console update itself either if hosted on a v2 or v3 rack.